### DISCUSSION TITLE
Why does Tree ignore the markup elements and the guide lines setting?

### DISCUSSION THREAD

**Discussion by yansklyarenko** [spectreconsole/spectre.console#378] [Q&A] [answered]
I have a very basic example of `Tree` usage, basically stolen from the docs:

```c#
var root = new Tree("Root").Guide(TreeGuide.BoldLine);

root.AddNode("[yellow]Foo[/]")
    .AddNode("[strikethrough blue]Qux[/]");
root.AddNode("[yellow]Bar[/]");

AnsiConsole.Render(root);

```
However, the output is not as expected:

![image](https://user-images.githubusercontent.com/3975939/115260778-b95db280-a13b-11eb-8672-b675e5ebef94.png)

As you can see, the colors are applied, but the `strikethrough` markup is ignored. Besides, it seems to ignore the guide line setting as well.

The version of Spectre.Console is 0.39.0. The version of PowerShell is 5.1.18362.1171. 

What am I missing? It must be something related to my env, 'cause it's very basic to be unnoticed all this time, but I can't figure out what's wrong. I would appreciate any hint.

**Comment by patriksvensson** [comment, marked as answer]
@yansklyarenko It doesn't consider your console to support Unicode characters or it thinks your terminal doesn't support ANSI escape sequences. You can override this behavior by setting the terminal encoding to UTF-8 explicitly.

The VT code for `strikethrough` is also not supported in all terminals.

What does `dotnet example info` say?

**Reply by yansklyarenko** [reply]
Thanks for the hint about `dotnet example info`! The terminal encoding indeed was different, and when I started the PowerShell like this: 
      `Powershell.exe -NoExit -Command "$OutputEncoding = [Console]::OutputEncoding = [Text.UTF8Encoding]::UTF8"` 
the lines became nice and solid. However, the `strikethrough` part is still not working.

The `dotnet example info` now says the following:

![image](https://user-images.githubusercontent.com/3975939/115285322-647b6580-a156-11eb-8a07-4ca175d1bdc9.png)

**Reply by patriksvensson** [reply]
Great!

There are no guarantees that text attributes like strikethrough or bold will be displayed since it's up to the terminal to support the corresponding VT codes. What terminal are you using?

**Reply by yansklyarenko** [reply]
I've just tried it in the Windows Terminal, and it works just fine. Not sure why ConEmu refused to render `strikethrough`, even after I applied the registry 'hack' [described here](https://stackoverflow.com/a/51681675/274535).

Anyway, I'm unblocked now. Thanks for your help, Patrik! 👍🏿

**Reply by yansklyarenko** [reply]
For others who might face a similar issue: the `strikethrough` support is still [an open feature request for ConEmu](https://github.com/Maximus5/ConEmu/issues/677).
